recap-of-techtalks-summer-edition-2021.md

```diff
@@ -0,0 +1,216 @@
+featured: true
+---
+
+Buckle up for the recap of another eclectic Techtalk.
+Lisa Tarpletti shared some tip on how to eco-design our system, Asma Oualmakran shed light on cybersecurity buzzwords, and Liza Morrison is walked us through a brand-new framework, Qwick.
+
+<!-- excerpt -->
+
+## Lisa Trapletti - Eco-designing its digital services. A path to sustainable IT
+
+{% image {src: "./blog/guest-blogs/images/techtalks-sophie.png", alt: "Sophie Ragas", widths: [300]} %}
+
+Lisa is a senior software developer working for iCure. She is certified in eco-design of digital services and passionate about the environmental transition and digital transformation.
+
+### The impact of IT
+
+Let’s kick off with some shocking numbers on the IT industry.
+
+Computing technologies are part of the daily life of 5 billions of users with 34 billions of connected devices, and it’s estimated that these numbers will double by 2025! 😱
+
+Also the environmental costs of IT’s ever-expanding presence are skyrocketing. Here are some figures:
+
+- The IT industry produces 4% of the greenhouse gas emissions in the world (more than the aviation sector which is responsible only for 2% of emissions)
+- It takes 10% of the world-wide electricity consumption
+- It is responsible for abiotic resources depletion
+- It perpetrates inhumane working conditions. In the Democratic Republic of Congo 6 million of workers died in coltan mines, a material used to produce electronics.
+
+Let’s break the IT industry into its component to understand why it has such a big ecological impact.There are three main actors in IT:
+
+1. The end user device
+2. The network infrastructure
+3. The data centres
+
+We can identify two sources of impact:
+
+1. The manufacture of equipments
+2. The manufacture of electricity
+
+Even if data centres conjure the image of the most energy-consuming part of IT industry, it is the end user devices that are responsible for most of the environmental impact - and especially their manufacture!
+
+{% image {src: "./blog/guest-blogs/images/techtalks-summer-2021-img1.pdf", alt: "a table showing that end user devices have by far the highest ecological impact (67,2%, compared to 12,3% of networks and 20,6% of data centres)", widths: [300,600,1000]} %}
+
+Of all end user devices the top 3 with the biggest environmental impact are the
+
+1. TV (14%)
+2. Desktop (7,5%)
+3. Laptop (7%)
+
+Maybe you too were expecting to see the smartphone showing up as number one? The reason why it doesn’t is that this ranking takes into account impact due to both manufacturing and use. With 4,5% the smartphone only comes sixth in the list but here’s the caveat: while it has a negligible impact in terms of usage (it doesn’t need to be plugged to energy to function) the smartphone has the highest impact for manufacturing only.
+
+### What can we do as citizen?
+
+As citizens we can strive towards a digital sobriety approach: try to buy less equipment which lasts longer and is really useful.
```

Comment:
This is a bit of a confusing sentence. Now it sounds like you should buy less equipment that lasts longer and is really useful. 

But I think you're trying to say that we should buy less equipment, but when we do it should focus on something that lasts longer and is really useful, no?